academic_helper/management/commands/fetch_courses.py

```diff
@@ -7,24 +7,32 @@
 from academic_helper.models import Course
 from academic_helper.utils.logger import log
 
-LIMIT = 50
-SKIP_EXISTING = True
+DEFAULT_SRC_FILE = "courses_2020.json"
+
+DEFAULT_LIMIT = 50
 SHUFFLE = True
 
 
-# TODO: Add arguments: src file, limit
 class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('--src_file', type=str, default=DEFAULT_SRC_FILE,
+                            help=f'Source file for courses (Default is {DEFAULT_SRC_FILE}).')
+        parser.add_argument('--limit', type=int, default=DEFAULT_LIMIT,
+                            help=f'Max number of courses to fetch (Default is {DEFAULT_LIMIT}).')
+        parser.add_argument('--fetch_existing', action='store_false', help='fetch existing courses.')
+
     def handle(self, *args, **options):
-        with open("courses_2020.json", encoding="utf8") as file:
+        with open(options['src_file'], encoding="utf8") as file:
             courses = json.load(file)
         if SHUFFLE:
             random.shuffle(courses)
         fail_count = 0
         for i, course in enumerate(courses):
-            if i > LIMIT:
+            if i > options['limit']:
                 break
             course_number = course["id"]
-            if SKIP_EXISTING:
+            if options['fetch_existing']:
```

Comment:
You confused true with false: rename `fetch_existing` to `skip_existing`, and set it's default to true.